relation/src/gadgets/emulated.rs

```diff
@@ -318,11 +318,6 @@ impl<F: PrimeField> PlonkCircuit<F> {
             E::NUM_LIMBS,
         );
 
-        // range checking for output c
```

Comment:
Add a comment on func header that we don't need to rangecheck a and c.

Reply:
added in 9345eea